code/modules/admin/verbs/adminhelp.dm

```diff
@@ -78,24 +78,46 @@
 		return
 	if(src.handle_spam_prevention(msg,MUTE_ADMINHELP))
 		return
+	var/ref_mob = "\ref[mob]"
+	for(var/datum/adminticket/T in admintickets)
+		if(T.permckey == src.ckey && T.resolved == "No")
```

Comment:
>src.
how many times do i have to say that "src.yourvar" and "yourvar" is the same aaaaaaaaaa